netbeans.apache.org/src/content/templates/footer.gsp

```diff
@@ -101,7 +101,7 @@
 <div class='footer-disclaimer'>
     <div class="footer-disclaimer-content">
         <p>Copyright &copy; 2017-2018 <a href="https://www.apache.org">The Apache Software Foundation</a>.</p>
-        <p>Licensed under the <a href="https://www.apache.org/licenses/">Apache Software License, version 2.0.</a></p>
+        <p>Licensed under the Apache Software <a href="https://www.apache.org/licenses/">license</a>, version 2.0</p>
```

Comment:
And not "Apache **Software** License, Version 2.0" but "Apache License, Version 2.0"?

Reply:
Well, of course. I need more sleep :-/

Reply:
The fact is that it's the "Apache Software License 1.1" (https://www.apache.org/licenses/LICENSE-1.1) or the "Apache License 2.0"  (https://www.apache.org/licenses/LICENSE-2.0). Let's remove the "Software" stuff. THanks @junichi11 !